### ISSUE TITLE
How do you deal with saves that have timestamps in the name?

### ISSUE THREAD

**Issue by cchulo** [mtkennerly/ludusavi#368]
### What's your question?

First off want to say excellent program, this is exactly what i needed to deal with syncing saves across all my devices.

I have a bit of an issue with the classic Resident Evil, the save files have timestamps in the name, i want to only keep two saves across my systems, if i keep back them up on one system, and then restore on the other, they end up accumulating: 
![image](https://github.com/user-attachments/assets/91c13647-32fc-41eb-a42f-7ad7e9afaf26)

**Comment by mtkennerly** [comment]
Yeah, that's tricky. Ludusavi doesn't handle this right now, but I can look into it. There was a somewhat similar issue here: https://github.com/mtkennerly/ludusavi/issues/253 

> First off want to say excellent program, this is exactly what i needed to deal with syncing saves across all my devices.

Thank you :D
